public/API/API_GetGameExtended.php

```diff
@@ -50,28 +50,118 @@
  *    string    Expiration                date the claim will expire
  */
 
+use App\Community\Models\AchievementSetClaim;
 use App\Platform\Enums\AchievementFlag;
+use App\Platform\Models\Achievement;
+use App\Platform\Models\Game;
+use Carbon\Carbon;
 
 $gameID = (int) request()->query('i');
 $flag = (int) request()->query('f', (string) AchievementFlag::OfficialCore);
-getGameMetadata($gameID, null, $achData, $gameData, flag: $flag, metrics: true);
 
-if ($gameData === null) {
+$game = Game::with('system')->find($gameID);
+$gameSetClaims = AchievementSetClaim::find($gameID);
```

Comment:
Does this query still run if `gameID` is 999999?

Reply:
Yes, it still runs when $game is null, I'll prevent that.